src/ppx_import.ml

```diff
@@ -288,6 +288,7 @@ let ptype_decl_of_ttype_decl ~manifest ~subst ptype_name
                  in
                  { pcd_name =
                      {txt = Ocaml_common.Ident.name cd.cd_id; loc = cd.cd_loc}
+                 ; pcd_vars = []
```

Comment:
I would expect `; pcd_vars = cd.cd_vars`?

Reply:
Hi @gasche, thanks for reviewing! The reason why I've made it an empty list is that in these patch PRs I simply preserve the behavior from before the AST bump. Sounds good to already implement the good new behavior though.

So it's a good point that here the good new behavior would probably be to use `cd.cd_vars`! The problem is that `cd` comes from the `Types` module, in which no `vars` field has been added to `constructor_declaration`. Btw, I was already wondering why no `vars` field has been added to `Types.constructor_declaration`. Do you know why (might be an obvious question)?

Reply:
Actually I don't know; I thought that this came from `Typedtree.constructor_declaration`, and that one has a `cd_vars` as expected. I'll ask on the original PR.